vars/buildfarm.yml

```diff
@@ -14,7 +14,7 @@ deployments:
   - name: "android-slave"
     containers:
     - name: "android-slave"
-      image: "docker.io/aerogear/jenkins-android-slave:dev"
+      image: "docker.io/aerogear/digger-android-slave-image:latest"
```

Comment:
for this repo, `latest` is probably ok. We should use the productised one in rhmap-ansible